.github/workflows/tests.yml

```
@@ -0,0 +1,68 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - "**" # all branches
+      - "!master" # except master
+      - "!develop" # except develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Cache npm dependencies
+        uses: actions/cache@v3
+        id: npm-cache
+        with:
+          path: |
+            - ./node_modules
+            - ./selenium/node_modules
+            - ./playwright/node_modules
+          key: npm-workspace-cache-${{ runner.os }}-${{ hashFiles('./**/package-lock.json') }}
+          restore-keys: |
+            npm-workspace-cache-${{ runner.os }}-
+      - name: Cache Maven dependencies
+        uses: actions/cache@v3
+        id: maven-cache
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: "8"
+          distribution: "temurin"
+      - name: Install Maven dependencies
+        # https://github.com/actions/cache#skipping-steps-based-on-cache-hit
+        if: steps.maven-cache.outputs.cache-hit != 'true'
+        run: mvn clean install -DskipTests
+      - name: Install NPM dependencies
+        # https://github.com/actions/cache#skipping-steps-based-on-cache-hit
+        if: steps.npm-cache.outputs.cache-hit != 'true'
+        run: npm --prefix=playwright ci && npm --prefix=selenium ci
+
+  playwright-tests:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Start fixture server
+        run: npm --prefix=playwright start &
+      - name: Run Playwright tests
+        run: mvn test -pl playwright
+      - name: Run Selenium tests
+        run: selenium/node_modules/axe-test-fixtures/fixtures && python2 -m SimpleHTTPServer 8001 & mvn test -q -pl selenium
+
+  selenium-tests:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Start fixture server
+        run: selenium/node_modules/axe-test-fixtures/fixtures &
```

Is `fixtures` a program?

---

[It doesn't look like it](https://github.com/dequelabs/axe-test-fixtures). I think this should be:

```suggestion
        run:  python2 -m SimpleHTTPServer 8001 &
        working-directory: selenium/node_modules/axe-test-fixtures/fixtures
```